Fedora-specific warnings about using nscd with sssd, and that
shared mappings don't show up in nscd -g statistics.

diff --git a/nscd/nscd.conf b/nscd/nscd.conf
index 3730835c50a349c4..b66faa3ca33b4b64 100644
--- a/nscd/nscd.conf
+++ b/nscd/nscd.conf
@@ -3,6 +3,9 @@
 #
 # An example Name Service Cache config file.  This file is needed by nscd.
 #
+# WARNING: Running nscd with a secondary caching service like sssd may lead to
+#          unexpected behaviour, especially with how long entries are cached.
+#
 # Legal entries are:
 #
 #	logfile			<file>
@@ -22,7 +25,12 @@
 #       suggested-size		<service> <prime number>
 #	check-files		<service> <yes|no>
 #	persistent		<service> <yes|no>
+#
 #	shared			<service> <yes|no>
+#	NOTE: Setting 'shared' to a value of 'yes' will accelerate the lookup
+#	      with the help of the client, but these lookups will not be
+#	      counted as cache hits i.e. 'nscd -g' may show '0%'.
+#
 #	max-db-size		<service> <number bytes>
 #	auto-propagate		<service> <yes|no>
 #
